### ISSUE TITLE
who is using

### ISSUE THREAD

**Issue by chunfz** [elsa-workflows/elsa-core#4859]
It is recommended to create a "Who is using" issue to discuss the usage of the elsa-workflow project.

**Comment by sfmskywalker** [comment]
Makes sense. Perhaps it should also include a "call to action" to anyone to mention if they are using Elsa. What do you think?
